server/routes/spec/tradesRouteSpec.js

```diff
@@ -6,35 +6,36 @@ var knex = require('knex');
 var config = require('../../db/knexfile');
 var app = require('../../index');
 
-var userArray = {
-  username: 'annaUser',
-  password: 'annaPassword',
-  name: 'anna',
-  email: 'anna@annars'
-};
-
-var trades = [{ user_id: 1, match_id: 1, symbol: 'GOOG', shares: 5, action: 'buy' },
-{ user_id: 1, match_id: 1, symbol: 'GOOG', shares: 5, action: 'sell' }];
-
-var matches = {
-  starting_funds: 100000,
-  startdate: 'Sat Dec 12 2015 16:55:38 GMT-0800 (PST)',
-  enddate: 'Fri Jan 29 2016 00:00:00 GMT-0800 (PST)',
-  status: 'in progress',
-  type: 'TEST'
-};
-
-var stock = {
-  name: 'Facebook, Inc.',
-  symbol: 'FB',
-  industry: 'Computer Software: Programming, Data Processing',
-  sector: 'Technology',
-  exchange: 'NASDAQ'
-};
-
-
  describe('/trades/:matchid/:userid', function () {
 
+  // ============= Test Data ============= \\
+
+  var userArray = {
+    username: 'annaUser',
+    password: 'annaPassword',
+    name: 'anna',
+    email: 'anna@annars'
+  };
```

Comment:
userArray could be userObject instead. This var naming can be confusing in the future